sambaplayersdk/src/main/java/com/sambatech/player/plugins/TrackingFactory.java

```diff
@@ -0,0 +1,13 @@
+package com.sambatech.player.plugins;
+
+class TrackingFactory {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS1118)